### ISSUE TITLE
🐛 Inconsisten emphasis/bolding on Personal information page

### ISSUE THREAD

**Issue by vd1992** [GCTC-NTGC/gc-digital-talent#10515]
## 🐛 Bug

Headers for profile information are not rendering in strong styling consistently. 
I think all of them are supposed to be bold 

## 📋 Steps to Reproduce

1. navigate to `/en/applicant/personal-information`
2. scroll down

## 📸 Screenshot

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/0539a2d4-673d-43fe-bde6-9a512488c2a8)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/4341c1cc-23c2-4784-9986-1f6a27572246)

## 🙋‍♀️ Proposed Solution

Consistency 

## ✅ Acceptance Criteria

- [ ] Criteria 1
- [ ] Criteria 2

**Comment by petertgiles** [comment]
@Jerryescandon could you please confirm all the field titles should have strong styling?

**Comment by vd1992** [comment]
I am a little confused as I also noticed that on the admin profile view, the bolding pattern is reversed? Intentionally?
`en/admin/users/:id/profile`

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/8daa5d5c-05fa-4940-9b42-8f78549e44bf)

**Comment by petertgiles** [comment]
> the bolding pattern is reversed? Intentionally?

I honestly don't remember and I haven't looked at the code [but from Figma](https://www.figma.com/design/XS4Ag6GWcgdq2dBlLzBkay/Admin-v3-(Profile-release)?node-id=465-2942&t=3IzyYKkN1xrJ2SdG-0), it seems like that's correct.  :shrug: 
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/86d244d3-c2ea-484d-8e14-2065e8cdde24)

**Comment by vd1992** [comment]
> > the bolding pattern is reversed? Intentionally?
> 
> I honestly don't remember and I haven't looked at the code [but from Figma](https://www.figma.com/design/XS4Ag6GWcgdq2dBlLzBkay/Admin-v3-(Profile-release)?node-id=465-2942&t=3IzyYKkN1xrJ2SdG-0), it seems like that's correct. 🤷
> 
> I thought they shared the same components...
> 

Yeah, that was what I thought so I was surprised they rendered opposite of one another
